server/methods/answers.js

```diff
@@ -1,186 +1,373 @@
 Meteor.methods({
-  createAnswer(answerData) {
-    var { description, questionId } = answerData;
-
-    var answer = {
-      ownerId: Meteor.userId(),
-      description: description,
-      upvotes: [],
-      downvotes: [],
-      createdAt: Date.now()
+    createAnswer(answerData) {
+        var {
+            description,
+            questionId,
+            questionOwnerEmail,
+            answerOwnerName
+        } = answerData;
+
+        var answer = {
+            ownerId: Meteor.userId(),
+            description: description,
+            upvotes: [],
+            downvotes: [],
+            createdAt: Date.now()
+        }
+
+        //Meteor.users.update({_id: answer.ownerId},{$inc: {answers: 1}});
+
+        // Add the new answer and get the id
+        var answerId = Answers.insert(answer);
+
+        // Append the answer to the question
+        Questions.update({
+            _id: questionId
+        }, {
+            $push: {
+                answers: answerId
+            }
+        });
+
+        let question = Questions.findOne({
+            _id: questionId
+        });
+        let icon = "<i class=\"idea icon\"></i>";
+        let content = "New answer on your question";
+        let link = `/discussions/${question.slug}`;
+
+        if (answer.ownerId != question.ownerId) {
```

Comment:
I know this problem is on us(we didn't have enough time) .. but it would be better/cleaner to move the notification logic to it's own module. 

> you don't have to do this is this PR though .. it's just a suggestion for the future